### ISSUE TITLE
Exception: `cweagans/composer-patches not found` on clean install

### ISSUE THREAD

**Issue by fredden** [sansecio/composer-integrity-plugin#10]
I have installed this plug-in, but I get an exception thrown saying `cweagans/composer-patches not found` when I run `composer integrity`.

# Steps to reproduce
1. `cd $(mktemp -d)`
1. `echo "{}" > composer.json`
1. `composer --no-interaction --no-plugins require --no-install sansec/composer-integrity-plugin`
   - Note that this gives me version `0.1.4` at time of writing.
1. `composer --no-interaction --no-plugins config allow-plugins.sansec/composer-integrity-plugin true`
1. `composer --no-interaction install`
1. `composer --no-interaction integrity`

Or, use this `composer.json` with `composer install && composer integrity`
```json
{
    "require": {
        "sansec/composer-integrity-plugin": "0.1.4"
    },
    "config": {
        "allow-plugins": {
            "sansec/composer-integrity-plugin": true
        }
    }
}
```

## Expected result
Report similar to that advertised in `README.md`

## Actual result
Exception `cweagans/composer-patches not found` is thrown.

![Screenshot_2023-04-25_09-41-48](https://user-images.githubusercontent.com/334786/234223142-836561b5-0654-44f7-b0b7-f76cb7b6b2fd.png)

Related: #7

**Comment by danslo** [comment]
CC @peterjaap 

I'm currently also refactoring how we do patches. For the time being you can use a lower version (0.1.3 should predate this).

**Comment by danslo** [comment]
This should be fixed in 0.1.5 now.
